R/post-tweet.R

```diff
@@ -227,3 +224,25 @@ wait_for_chunked_media <- function(resp, media_id, token = NULL) {
 
   invisible()
 }
+
+check_media <- function(media, alt_text) {
+  media_type <- tools::file_ext(media)
+  if (length(media) > 4) {
+    stop("At most 4 images per plot can be uploaded.", call. = FALSE)
```

Comment:
Now that we import rlang, you could simplify these to 

```suggestion
    abort("At most 4 images per plot can be uploaded.")
```

Reply:
We don't use classes of errors so I'll stick with base stop.

Reply:
`abort()` also gives you a better default for `call.` (so you don't forget) and generates more informative tracebacks.